### ISSUE TITLE
[CR 110] remove Block third-party cookies in Private setting

### ISSUE THREAD

**Issue by rebron** [brave/brave-browser#28176]
## Description 

Remove "Block third-party cookies setting in Private" as it's unnecessary and conflicts with Shields.
See: brave://settings/cookies

## Actual result:

<img width="698" alt="Screenshot 2023-01-31 at 10 50 07 AM" src="https://user-images.githubusercontent.com/5951041/215867573-5b96ed6c-f6c6-403c-83aa-63e76c62950d.png">

## Expected result:
<img width="692" alt="Screenshot 2023-01-31 at 11 52 32 AM" src="https://user-images.githubusercontent.com/5951041/215867764-0952f235-7ff8-4f41-b252-646e859c98a3.png">

## Reproduces how often: 

Easily.

## Brave version (brave://version info)

1.49.x with Chromium 110

## Version/Channel Information:

- Can you reproduce this issue with the current release? n/a
- Can you reproduce this issue with the beta channel? n/a
- Can you reproduce this issue with the nightly channel? yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? n/a
- Does the issue resolve itself when disabling Brave Rewards? n/a
- Is the issue reproducible on the latest version of Chrome? yes

**Comment by bridiver** [comment]
I don't understand, how does this conflict with shields? This is the default setting that we use.

**Comment by emerick** [comment]
I think there's a typo in this issue @rebron. It should be referencing the second radio button labelled "Block third-party cookies in Private".

**Comment by bridiver** [comment]
Do you mean remove "block third party cookies in private"?

**Comment by rebron** [comment]
Correct. Block third party cookies in private setting.

**Comment by kjozwiak** [comment]
We'll move back into `1.48.x` once https://github.com/brave/brave-core/pull/16884 has been merged into `1.48.x`.

**Comment by kjozwiak** [comment]
The above requires `1.48.153` or higher for `1.48.x` verification 👍

**Comment by stephendonner** [comment]
@brave/qa-team this is blocked by https://github.com/brave/brave-browser/issues/28261 👍

**Comment by LaurenWags** [comment]
Removed `QA/Blocked` as this can be verified in `1.48.155` and higher where https://github.com/brave/brave-browser/issues/28261 is included.

**Comment by LaurenWags** [comment]
Verified with

```
Brave | 1.48.155 Chromium: 110.0.5481.77 (Official Build) (x86_64)
-- | --
Revision | 65ed616c6e8ee3fe0ad64fe83796c020644d42af-refs/branch-heads/5481@{#839}
OS | macOS Version 12.6.3 (Build 21G419)
```

Confirmed `Block third-party cookies in Private` setting is no longer displayed under brave://settings/cookies.

<img width="1348" alt="1 48 x" src="https://user-images.githubusercontent.com/28145373/216618336-3d6c7b86-4e6e-49b3-a6e0-6109111c3932.png">

**Comment by LaurenWags** [comment]
Removed `QA/Test-All-Platforms` label per discussion with @kjozwiak @rebron but we will want to verify this one on Windows.

**Comment by MadhaviSeelam** [comment]
Verification `PASSED` with
```
Brave | 1.48.155 Chromium: 110.0.5481.77 (Official Build) (64-bit)
-- | --
Revision | 65ed616c6e8ee3fe0ad64fe83796c020644d42af-refs/branch-heads/5481@{#839}
OS | Windows 11 Version 21H2 (Build 22000.1455)
```

Install 1.48.155
launch Brave
open `brave://settings/cookies`

#### Confirmed `Block third-party cookies in Private setting` is no longer displayed.


<img width="947" alt="image" src="https://user-images.githubusercontent.com/98358127/216687723-423ad434-1f79-4e83-aab5-2eadb2ab0832.png">
